src/wovnio/wovnphp/Utils.php

```diff
@@ -8,7 +8,14 @@ class Utils
     // will return the store and headers objects
     public static function getStoreAndHeaders(&$env)
     {
-        $file = isset($env['WOVN_CONFIG']) ? $env['WOVN_CONFIG'] : DIRNAME(__FILE__) . '/../../../../wovn.ini';
+        if (isset($_SERVER['WOVN_CONFIG'])) {
+            $file = $_SERVER['WOVN_CONFIG'] ? $_SERVER['WOVN_CONFIG'] : DIRNAME(__FILE__) . '/../../../../wovn.json';
```

Comment:
I guess this line should use `$env`.

Reply:
Yep. Just curious: why do we need to "rename" $_SERVER into $env here? It seems that $env here is purely an alias for $_SERVER anyways.

Reply:
Yes.
I guess there are two purpose.
- Change $_SERVER inside method. PHP can pass reference by adding `&` to the arguments.
- Make it easy to test, because test don't need to care about $_SERVER and just check arguments.